src/linter.rs

```diff
@@ -212,6 +214,9 @@ pub fn lint_path(
         let (transformed, fixed, messages) = lint_fix(&contents, path, package, settings)?;
         if fixed > 0 {
             if matches!(autofix, fixer::Mode::Apply) {
+                if level >= LogLevel::Default {
+                    eprintln!("Fixing {} error(s) in {}", fixed, fs::relativize_path(path));
```

Comment:
Are we going with “violation(s)” here too?

Reply:
Eventually, but for now I wanted it to be consistent with the rest of the output (which says "error(s)" at the bottom).

Also: we may continue to say "error" in both places, because an error will be a severity level for a violation (so, in theory, we could say "X errors and Y warnings").